components/src/CartCreate/cart.create.less

```diff
@@ -0,0 +1,263 @@
+/**
+ * Copyright © 2019 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+@import './../../../app/src/theme/common.less';
+
+.cart-create-modal {
+  min-width: ~'calc(100vw - 4.8rem)';
+
+  @media screen and (min-width: 681px) {
+    min-width: 580px;
+  }
+
+  .modal-content {
+    border: none;
+
+    @media screen and (min-width: 681px) {
+      padding: 0 15px;
+    }
+  }
+
+  .ep-btn {
+    text-transform: none;
+  }
+
+  .create-cart-btn {
+    min-height: 44px;
+    background: #F2F6FA;
+
+    @media screen and (min-width: 681px) {
+      min-width: 315px;
+    }
+  }
+
+  .btn-container {
+    text-align: right;
+
+    .ep-btn {
+      min-height: 34px;
+    }
+
+    .cancel-btn {
+      width: 120px;
+      border-color: transparent;
+    }
+
+    .save-btn {
+      width: 120px;
+      margin-right: 0;
+
+      .miniLoader {
+        width: 20px;
+        height: 20px !important;
+        display: inline-block;
+        vertical-align: middle;
+        margin: 0;
+        border-width: 2px;
+        border-top-width: 5px;
+      }
+    }
+  }
+
+  + button[class*=styles_closeButton] {
+    top: 22px;
+    right: 22px;
+    cursor: pointer;
+    [class*=styles_closeIcon] {
+      height: 24px;
+      width: 24px;
+      fill: #565656;
+    }
+  }
+
+  .edit-mode-form {
+    width: 316px;
+
+    label {
+      font-size: 12px;
+      font-weight: 600;
+      color: #777777;
+      margin-bottom: 2px;
+    }
+
+    .cart-edit-field {
+      height: 34px;
+      width: 100%;
+      padding: 0 30px 0 15px;
+      border: 1px solid #D4D3D3;
+      color: #1F2532;
+      border-radius: 4px;
+
+      &:focus {
+        border-color: rgba(0,153,222,0.5);
+        box-shadow: 1px 1px 2px 0 rgba(0,153,222,0.12);
+        outline: 0 auto transparent;
+      }
+
+      &-wrap {
+        position: relative;
+      }
+    }
+
+    .clear-field-btn {
+      background: #DBDBDB;
+
+      &:hover {
+        background: #c5c5c5;
+      }
+    }
+  }
+}
+
+.create-cart-btn-wrap {
+  padding: 5px 0 15px;
+  border-bottom: 1px solid #CCCCCC;
```

Comment:
There are quite a few colors here that we need to replace with their vars from common.less